contracts/scripts/common/Context.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {CommonBase} from "forge-std/Base.sol";
+
+abstract contract Context is CommonBase {
```

Comment:
reason for moving this to a separate file?

Reply:
`DeployBase` and `TestUtils` have them causing conflicts.